tests/acceptance/pageObjects/FilesPageElement/filesList.js

```diff
@@ -484,7 +484,9 @@ module.exports = {
               return
             }
 
-            scrollDistance += virtualScrollWrapper.clientHeight
+            const listItemHeight = document.querySelector(listItemSelector).clientHeight
+
+            scrollDistance += listItemHeight * 5
```

Comment:
This will make the function slightly slower but it will make it much more stable. Some rows in the files list were skipped when doing a scroll with the visible height of the table. This should ensure that the row is always found (if it exists of course 🙂 )

Reply:
Nice! However, the `scroll item to the top of the list` didn't work. I removed it, since I don't see a reason why we need to scroll it back to the top of the list. If there is a reason, please elaborate. :-)

Reply:
Sometimes can happen that the actions dropdown is cut off (either top or bottom part) and selectors are then failing. Happening in small resolution tests. If it would be scrolled to the top we'd have then consistent behaviour on all resolutions and no menu would be cut off.

Reply:
Ah ok, thanks for the explanation. Might be best then to revert my commit and fix the `scroll item to top`, which was not working.

Reply:
iOS tests are still failing, so this is probably what you wanted to achieve...

Reply:
> iOS tests are still failing, so this is probably what you wanted to achieve...

Exactly. I'll try to debug why the scroll was failing.

Reply:
Reverted all new scrolling magic and skipped the tests on iPhone instead until we have a fix for the actions dropdown.